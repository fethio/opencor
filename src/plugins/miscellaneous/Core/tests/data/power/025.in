<math xmlns="http://www.w3.org/1998/Math/MathML">
    <apply>
        <eq/>
        <ci>a</ci>
        <apply>
            <power/>
            <apply>
                <minus/>
                <apply>
                    <power/>
                    <ci>b</ci>
                    <ci>c</ci>
                </apply>
            </apply>
            <ci>d</ci>
        </apply>
    </apply>
</math>
