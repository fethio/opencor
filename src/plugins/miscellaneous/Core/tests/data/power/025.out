<math xmlns="http://www.w3.org/1998/Math/MathML">
    <mrow>
        <mi mathvariant="italic">a</mi>
        <mo>=</mo>
        <msup>
            <mrow>
                <mo>(</mo>
                <mo>âˆ’</mo>
                <msup>
                    <mi mathvariant="italic">b</mi>
                    <mi mathvariant="italic">c</mi>
                </msup>
                <mo>)</mo>
            </mrow>
            <mi mathvariant="italic">d</mi>
        </msup>
    </mrow>
</math>
